sdk/communication/Azure.Communication.Chat/src/Models/AzureCommunicationChatServiceRestClient.cs

```diff
@@ -5,6 +5,6 @@
 
 namespace Azure.Communication.Chat
 {
-    [CodeGenClient("ServiceRestClient")]
+    [CodeGenClient("AzureCommunicationChatServiceRestClient")]
```

Comment:
Is this a correct title?

Reply:
I am guessing it is removing the whitespaces in the title.
Eg: "Azure Communication Chat Service" is becoming "AzureCommunicationChatServiceRestClient". In which case, I wonder if we should set the titles differently.
https://github.com/Azure/azure-rest-api-specs/blob/master/specification/communication/data-plane/Microsoft.CommunicationServicesChat/preview/2020-09-21-preview2/communicationserviceschat.json#L4

Reply:
@pakrym Yes, it is coming from the title. Please let me know if we want to set a different title for this.